test/run-tests

```diff
@@ -296,6 +315,7 @@ def fetch_image(url, cache, label):
             return None
 
         os.setxattr(image_tempfile.name, URL_XATTR, os.fsencode(url))
+        os.setxattr(image_tempfile.name, DATE_XATTR, image_last_modified_by_src)
```

Comment:
what is the format for `image_last_modified_by_src`?  Do you have to use `os.fsencode` to encode the value before storing it in the xattr?

Reply:
The `image_last_modified_by_src` variable stores strings like "Wed, 22 Apr 2020 12:24:13 GMT". It has `str` filetype so in the latest commit I added `os.fsencode` to convert it to bytes-like object.

The downloaded image file metadata looks like:
$ getfattr -d CentOS-7-x86_64-GenericCloud.qcow2c
# file: CentOS-7-x86_64-GenericCloud.qcow2c
user.xdg.origin.url="https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud.qcow2c"
user.dublincore.date="Wed, 22 Apr 2020 12:24:13 GMT"